deployment/v3/external/kafka/README.md

```diff
@@ -9,3 +9,76 @@ Kafa is required only if you are installing Registration Processor.
 ```
 $ ./install.sh
 ```
+
+## Procedure to Backup & Restore bitnami kafka via Velero
+
+#### Prerequisites
+* Required separate `minio` server / 'S3' , accessible from both backup & restore clusters along with `api` & `console` port accessible.
+* Ensure to create a bucket with name `velero`.
+* Required backup & restore servers cluster config file.
+* Install [velero](https://velero.io/docs/v1.9/basic-install/#install-the-cli) cli on both cluster management machines.
+* Install [minio client](https://docs.min.io/docs/minio-client-quickstart-guide.html) cli on both cluster management machines.
+
+#### Backup kafka
+* Login to the console machine,navigate to the `Kafka directory (under /v3/external/)`, and execute the backup script to create a backup of Kafka.
```

Comment:
Login to the console machine, navigate to..............

Space after comma needs to be added in the above sentence